Makefile

```diff
@@ -179,6 +182,7 @@ capella/base: base
 		--build-arg BUILD_TYPE=$(CAPELLA_BUILD_TYPE) \
 		--build-arg CAPELLA_VERSION=$$CAPELLA_VERSION \
 		--build-arg "CAPELLA_DROPINS=$(CAPELLA_DROPINS)" \
+		--build-arg "INJECT_PACKAGES=$(INJECT_LIBS_CAPELLA)" \
```

Comment:
Wouldn't it be easier if the Makefile and Dockerfile args have the same name?

Also, I do not see where this new arg is used.

Reply:
> Wouldn't it be easier if the Makefile and Dockerfile args have the same name?

This is inspired by the `BUILD_TYPE` variable, which is used in multiple different images and has different names in the Makefile. To allow a fine-grained control over the `INJECT_PACKAGES` flag for each image build, there is a `INJECT_PACKAGES` flag for each image (e.g. `INJECT_LIBS_CAPELLA`). Right now, we just have one flag, but there may be more images in the future.

> Also, I do not see where this new arg is used.

You can use it by setting the `INJECT_LIBS_CAPELLA` environment variable. By default it's `false` (for environments with full internet access), but you can also set it to `true` to test the restricted environment setup (which is also used in the Gitlab CI/CD pipeline).